eq_translations/extractable_strings.py

```diff
@@ -117,22 +117,22 @@
     },
     {
         "json_path": "$..question.definitions[*].title",
-        "description": "Question definition link",
+        "description": "Question definition (array type) link",
```

Comment:
(array type) is redundant and has no meaning to translators, why do we need it?

Reply:
I was just trying to make the descriptions unique due to Petes comment above. Thought this was the best way of describing the different versions of definitions